www/templates/survey/enketo/notes_list.html

```diff
@@ -1,9 +1,12 @@
 <div>
     <ion-item class="notes-list" style="background-color: transparent">
-        <ul>
-            <li class="notes-list-entry" ng-repeat="entry in additionEntries | orderBy: ['data.start_ts', 'data.end_ts']">
+        <ul class="notes-list-grid">
+            <li ng-repeat="entry in additionEntries | orderBy: ['data.start_ts', 'data.end_ts']">
                 <b class="two-lines" ng-click="editEntry(entry)">{{entry.data.label}}</b>
-                <i ng-click="editEntry(entry)">{{entry.displayTime || setDisplayTime(entry)}}</i>
+                <i ng-click="editEntry(entry)">
+                    <p>{{ entry.displayDt.date }}</p>
+                    <p>{{ entry.displayDt.time || setDisplayDt(entry).time }}</p>
```

Comment:
maybe I should have noticed this earlier, but I don't quite understand what 
`{{ entry.displayDt.time || setDisplayDt(entry).time }}` does

It looks like it lazily populates the displayDt, but then why don't you need it for

`{{ entry.displayDt.date }}` as well?

Is it because the date is only displayed for multi-day cards while the time is always displayed? And when we call `setDisplayDt`, the date and time will both be displayed.

I think that is it; but you might want to add a comment to make it easier for others to read the code in the future.

Reply:
yes, that's why. Will add comment